mapper.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 var (
-	metricLineRE      = regexp.MustCompile(`^(\*\.|[^*.]+\.)*(\*|[^*.]+)$`)
+	metricLineRE      = regexp.MustCompile(`^(\*\.|[^*.]+\.|\.)*(\*|[^*.]+)$`)
```

Comment:
Turns out in graphite one could also not specify the type in addition to group, resulting in a metric ..a.b.c.

Since lookback/ahead are not supported in go, used string manipulation to address the issue.